.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -274,9 +273,12 @@ jobs:
           env | grep '^GITHUB' > "/tmp/github_env_${GITHUB_RUN_ID}"
       - name: Test PyTorch
         run: |
-          if [[ $NUM_TEST_SHARDS -eq 2 ]]; then
-            export SHARD_NUMBER=$TEST_CONFIG
+          if [[ $TEST_CONFIG == 'multigpu' ]]; then
+            TEST_COMMAND=.jenkins/pytorch/multigpu-test.sh
```

Comment:
This might've been far fetch. But can we do the workflow template based on the input? 
for example, if `test_runner_type` is linux.2xlarge, there's no point checking `multigpu`, this code right now is allowing running multigpu-test.sh on a 1-GPU instance, yes?

Reply:
Oh shoot you're totally right! Yeah, currently this is using the same runner for the multigpu tests as for the others; let me fix that

Reply:
@seemethere I'm going to add a label for `g3.16xlarge` in `.github/scale-config.yml`; do I need to also add support for that somewhere else? I don't know how our autoscaling infra works

Reply:
what should I set for `max_available` btw?

Reply:
also, does the `g3.16xlarge` have the same disk size as the `g3.8xlarge` (150 gigabytes)? I don't see that info given here: https://aws.amazon.com/ec2/instance-types/

Reply:
oh wait, if this config is pulled from `master`, does that mean I need to first merge another PR adding the label for `g3.16xlarge` before I can fully test and merge this PR?

Reply:
opened #60888 for this